lib/workers/pr/index.ts

```diff
@@ -110,7 +111,15 @@ export async function ensurePr(
     config.branchName
   ];
   // Check if existing PR exists
-  const existingPr = await platform.getBranchPr(branchName);
+  let existingPr = await platform.getBranchPr(branchName);
+  if (!existingPr) {
+    existingPr = await findPr({
```

Comment:
We need to make sure this searches from most recent first, in case multiple of the same PR title/branch name exist.